### PR TITLE
Update FreeBayes wrapper to 1.3.1

### PR DESCRIPTION
**Pull request by davebx**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR updates an existing tool or tool collection

This is a clean version of #2487, since my attempts at rebasing that one ended up with one merge conflict after another.

### CODE REVIEW

**Comment by bgruening** [comment]
@davebx I'm not a fan of dumping the parameters into the help. The parameters should be explained as part of the UI. But if you like it keep it we do not have rules for it atm.


@bernt-matthias do you have time to look at it?

**Comment by bernt-matthias** [comment]
Hi @bgruening. Very busy the next two weeks. Could have a look afterwards. Have my points already been considered. The comments seems still to be pending.

**Comment by nekrut** [comment]
I disagree vigorously with the fact that we do not need to provide parameters in help. I think it is vital. The complex tools we hide many parameters. Having them all listed is a way to let users scan the parameters and decide what they may or may not change. It also provides continuity with the command line version of the tool. If this is the only issue, please accept this PR.

**Comment by bgruening** [comment]
@nekrut I'm fine with merging. The help is not a stopper I think we don't have a rule for that afaik. 
But if your only argument is that "a list of all parameters is needed", then this list should be generated automatically and we should have a UI for that - to fulfill your use-case - a coherent one for all tools.
I don't see a point for duplicating information twice. And one of them in an unstructured way.
This will cause issues in the long run imho, as we need to update two places etc ...

**Comment by nekrut** [comment]
I don't think we duplicate this info. We identity an option right by the UI widget, but then also provide a global list of parameters in help. This way a user can see all parameters at once. I often use this approach. I get lost if a tool section does not list all options.

File: tools/freebayes/macros.xml [hunk 1]
```diff
@@ -44,9 +44,9 @@
         </conditional>
     </xml>
     <token name="@COVERAGE@">
-        --min-coverage ${options_type.min_coverage}
-        --skip-coverage ${options_type.skip_coverage}
-        --limit-coverage ${options_type.limit_coverage}
+        --min-coverage ${coverage_options.min_coverage}
```
Comment (bgruening) [inline tools/freebayes/macros.xml]: the test failed beacuse `--min-coverage 0 --skip-coverage 0 --limit-coverage 400` is not part of test 5 ... is that correct?

Comment (davebx) [reply tools/freebayes/macros.xml]: It should be passing, 0 is the default for those options when they're not specified, and test 5 is passing on my local system.

Comment (bgruening) [reply tools/freebayes/macros.xml]: The output is differes slightly ... maybe you need to commit your test-data? It is failing atm.

**Comment by bgruening** [comment]
Merci!

**Pull request merged**